core/src/test/scala/unit/kafka/utils/TestUtils.scala

```diff
@@ -1503,6 +1503,36 @@ object TestUtils extends Logging {
     out.toString
   }
 
+  /**
+   * Capture the console error during the execution of the provided function.
+   */
+  def grabConsoleError(f: => Unit) : String = {
+    val err = new ByteArrayOutputStream
+    try scala.Console.withErr(err)(f)
+    finally scala.Console.err.flush
+    err.toString
+  }
+
+  /**
+   * Capture both the console output and console error during the execution of the provided function.
+   */
+  def grabConsoleOutputAndError(f: => Unit) : (String, String) = {
+    val out = new ByteArrayOutputStream
+    val err = new ByteArrayOutputStream
+    try {
+      scala.Console.withOut(out) {
+        scala.Console.withErr(err) {
+          f
```

Comment:
nit: This would look a little less odd if we made it a normal function and invoked with parenthesis.